src/Session.php

```diff
@@ -332,6 +332,10 @@ public function executeScript($script)
     /**
      * Execute JS in browser and return it's response.
      *
+     * The "return" keyword is optional in the script passed as argument.
```

Comment:
It would be nice to have an actual example.

Reply:
see https://github.com/minkphp/driver-testsuite/blob/master/tests/Js/JavascriptEvaluationTest.php for the tests